com.jetbrains.CLion.yml

```diff
@@ -0,0 +1,63 @@
+app-id: com.jetbrains.CLion
+branch: stable
```

Comment:
```suggestion
```
it's the default already

Reply:
I'm not familiar with Flatpak and/or Flathub standards, but shouldn't it be good to be more explicit?

Reply:
done